client/app/dbaas/logs/detail/logs-detail.less

```diff
@@ -27,3 +27,9 @@
     display: none;
   }
 }
+
+.alerts {
+  .oui-dropdown__trigger {
+    max-width: 34rem;
```

Comment:
Could bootstrap utilities suit your needs here ? https://getbootstrap.com/docs/4.3/utilities/sizing/#relative-to-the-parent

Reply:
@marie-j Thanks for your comment. I tried your tip without success.

Reply:
Using a `<oui-select>` instead of a `<oui-action-menu>` will prevent you to add theses specifics rules.

Reply:
Thanks for your suggestion, i changed for <oui-select>.